tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -0,0 +1,287 @@
+#!/bin/bash
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+[ -z "${DEBUG}" ] || set -x
+set -o errexit
+set -o nounset
+set -o pipefail
+
+readonly project="kata-containers"
+
+readonly script_name="$(basename "${BASH_SOURCE[0]}")"
+readonly script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+readonly prefix="/opt/kata"
+readonly repo_root_dir="$(cd "${script_dir}/../../../.." && pwd)"
+readonly version_file="${repo_root_dir}/VERSION"
+readonly versions_yaml="${repo_root_dir}/versions.yaml"
+
+readonly clh_builder="${repo_root_dir}/tools/packaging/static-build/cloud-hypervisor/build-static-clh.sh"
+readonly firecracker_builder="${repo_root_dir}/tools/packaging/static-build/firecracker/build-static-firecracker.sh"
+readonly kernel_builder="${repo_root_dir}/tools/packaging/kernel/build-kernel.sh"
+readonly qemu_builder="${repo_root_dir}/tools/packaging/static-build/qemu/build-static-qemu.sh"
```

Comment:
Might be worth creating a variable for the common `"${repo_root_dir}/tools/packaging/static-build/` prefix to make all these lines shorter / easier to read.

Reply:
+1

Reply:
+1